lib/workers/repository/update/pr/changelog/source.ts

```diff
@@ -185,11 +188,19 @@ export abstract class ChangeLogSource {
   private findTagOfRelease(
     version: allVersioning.VersioningApi,
     packageName: string,
+    depName: string,
     depNewVersion: string,
     tags: string[],
   ): string | undefined {
-    const regex = regEx(`(?:${packageName}|release)[@-]`, undefined, false);
-    const exactReleaseRegex = regEx(`${packageName}[@\\-_]v?${depNewVersion}`);
+    const releaseNameRegexPrefix = `(?:${depName}|${packageName}|release)[@_-]`
```

Comment:
Simplified a bit here. Had 2 questions:

1. Should/can this prefix include the optional `v?`?
1. Naming is hard—do you suggest a better one? (`releaseRegexPrefix`?)

Example:
```suggestion
    const releaseRegexPrefix = `(?:${depName}|${packageName}|release)[@_-]v?`
```

This shouldn't negatively effect tag filtering.